presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -0,0 +1,45 @@
+=========================
+T-Digest Functions
+=========================
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile
+information.  The presto type for this data structure is called ``tdigest``.
+T-digests may be merged without losing precision, and for storage
```

Comment:
`without losing precision` -> that's not accurate. Every time an element is added to a t-digest, precision is sacrificed *somewhere*.

Reply:
What do you suggest?

Reply:
I'd just remove that qualifier and leave it at "T-digests may be merged"